packages/synapse-react-client/src/components/SynapseChat/SynapseChatInteraction.tsx

```diff
@@ -0,0 +1,57 @@
+import React, { useMemo } from 'react'
+import { ListItem, ListItemText, Skeleton } from '@mui/material'
+import { useTheme } from '@mui/material'
+import { ColorPartial } from '@mui/material/styles/createPalette'
+import { AgentChatRequest } from '@sage-bionetworks/synapse-types'
+import { useGetAgentChatWithAsyncStatus } from 'src/synapse-queries/chat/useChat'
+
+export type SynapseChatInteractionProps = {
+  sessionId: string
+  userMessage: string
+}
+
+export const SynapseChatInteraction: React.FunctionComponent<
+  SynapseChatInteractionProps
+> = ({ sessionId, userMessage }) => {
+  const theme = useTheme()
+  const agentChatRequest: AgentChatRequest = useMemo(() => {
+    return {
+      chatText: userMessage,
+      sessionId: sessionId,
+    }
+  }, [userMessage])
+  const { data: chatResponse, isLoading } =
+    useGetAgentChatWithAsyncStatus(agentChatRequest)
```

Comment:
mentioned elsewhere, this should probably be a mutation (you might need to move this to the outer component)